src/cpp/session/modules/SessionThemes.R

```diff
@@ -1027,47 +1014,114 @@
          globalPath <- normalizePath(.rs.getThemeInstallDir(TRUE), mustWork = FALSE, winslash = "/")
          if (identical(file.path(globalPath, justFileName), actualPath))
          {
-            stop(
-               "Unable to remove the specified theme \"",
-               name,
-               "\", which is installed for all users. Please contact your system administrator.")
+            stop("The specified theme is installed for all users. Please contact your system administrator to remove the theme.")
          }
          else
          {
-            stop(
-               "Unable to remove the specified theme \"",
-               name,
-               "\". Please check your file system permissions.")
+            stop("Please check your file system permissions.")
          }
       }
    }
 })
 
+# C++ Wrappers =====================================================================================
+.rs.addFunction("convertThemeForCpp", convertThemeForCpp <- function(themePath) {
```

Comment:
If I don't name them, how do I call them from C++? Also because I couldn't think of a better distinct name for these wrappers that are intended to be called from C++ (I realise now the comment may be confusing - it's meant to mean "wrappers of the R code to be called from C++).